uui/components/pickers/PickerModal.tsx

```diff
@@ -54,9 +54,12 @@ export function PickerModal<TItem, TId>(props: PickerModalProps<TItem, TId>) {
 
     const renderFooter = () => {
         const hasSelection = view.getSelectedRowsCount() > 0;
+        const rowsCount = view.getListProps().rowsCount;
+        const isEmptyRowsAndHasNoSelection = (rowsCount === 0 && !hasSelection);
+
         return (
             <>
-                {view.selectAll && (
+                {view.selectAll && !isEmptyRowsAndHasNoSelection && (
```

Comment:
Let's better disable this button, rather than hide it. You can do it via `isDisabled` prop of `LinkButton`

Reply:
got it